### ISSUE TITLE
STEP files for CAD

### ISSUE THREAD

**Issue by apoorv1in** [Annex-Engineering/Gasherbrum-K3#44]
Is there a plan to release the CAD files in Step format or a format that is widely used?
It is a big letdown seeing x_t file format, I believe most of diy community uses step format which can be opened and edited in most of opensource CAD software.
Kindly consider this suggestion, If you really want the design to be mass adopted. Take the hint from Voron, I am not asking for detailed documentation just the CAD

**Comment by whattheflukz** [comment]
Skill issue, Onshape can import X_T

**Comment by Trails5000** [comment]
flukz closed this but im going to expand on it, because  you probably dont have a design background where your daily into and out of CAD.

STEP is considered to be a lossy format, it tends to randomly lose surfaces and require alot of repair upon import. This is something that fusion ignores. It also doesnt help that most of the CAD companies cant seem to agree on standards for step, what is exported out of solidworks is not the same as what is exported out of fusion.

Therefore, we share "native engine cad" only. Most of the dev team works in solidworks, which is powered by the parasolid engine. X_T files are basically native files with the file tree and all personally identifying data stripped out.

TLDR: step sucks, move onto a better format.
